tests/e2e/scenarios/node-remove.go

```diff
@@ -133,6 +133,14 @@ func defineNodeRemovalTest(driver *baremetalDriver) {
 		_, err = framework.NodeHasTaint(f.ClientSet, taintNodeName, &taint)
 		framework.ExpectNoError(err)
 
+		// wait until csibmnode labeled with node.dell.com/drain=drain
+		for start := time.Now(); time.Since(start) < time.Minute*10; time.Sleep(time.Second * 10) {
```

Comment:
Can you attach logs with the passed test to confirm its stability?

Reply:
> Can you attach logs with the passed test to confirm its stability?

Have attached logs in the testing part of this PR's description.